### ISSUE TITLE
Cannot set property 'validating' of undefined

### ISSUE THREAD

**Issue by myagoo** [final-form/final-form#407]
### Are you submitting a **bug report** or a **feature request**?

A bug report
### What is the current behavior?

```js
  const {submitting, validating} = useFormState({
    subscription: {
      submitting: true,
      validating: true,
    },
  })
```

trigger an exception when clicking submit button

`TypeError: Cannot set property 'validating' of undefined
    at final-form.es.js:42
`

### What is the expected behavior?

useFormState should return validating state without crashing

### What's your environment?

The problem appeared with the 4.20.2 version, it is not present with any older version

**Comment by kosiakk** [comment]
This seems to be related to change 700a3ac0437dbec030dcf702918b86a3ef141ff5, and is also reported in https://github.com/final-form/react-final-form/issues/904

**Comment by GarvitRajput** [comment]
Any update here?

**Comment by xuedafei** [comment]
> Any update here?
install version 4.20.1

**Comment by gertdreyer** [comment]
[Fixed in v4.20.3](https://github.com/final-form/react-final-form/issues/904#issuecomment-930056241)
